feat: add stripe-cli fixtures for creating products and prices (#58)
vercel/nextjs-subscription-payments: README.md

  - Price 1: 20 USD per month
  - Price 2: 200 USD per year

> #### Doing this automatically with fixtures

Reviewer: ```suggestion
#### Generate test data with the Stripe CLI
```

> The Stripe cli `fixtures` command lets you run a json file as a series of API requests. We have provided you with a [fixtures file](fixtures/stripe-fixtures.json) that you can use to automatically bootstrap some products and prices in your Stripe account. This can be run with `stripe fixtures fixtures/stripe-fixtures.json`.

Reviewer: ```suggestion
The [Stripe CLI](https://stripe.com/docs/stripe-cli#install) `fixtures` command executes a series of API requests defined in a JSON file. Top help speed up the setup, we have added a [fixtures file](fixtures/stripe-fixtures.json) to bootstrap test product and pricing data in your Stripe account. Simply run `stripe fixtures fixtures/stripe-fixtures.json`.
```

  - Price 1: 20 USD per month
  - Price 2: 200 USD per year

#### Generate test data with the Stripe CLI

> The [Stripe CLI](https://stripe.com/docs/stripe-cli#install) `fixtures` command executes a series of API requests defined in a JSON file. Top help speed up the setup, we have added a [fixtures file](fixtures/stripe-fixtures.json) to bootstrap test product and pricing data in your Stripe account. Simply run `stripe fixtures fixtures/stripe-fixtures.json`.

Reviewer 2: ```suggestion
The [Stripe CLI](https://stripe.com/docs/stripe-cli#install) `fixtures` command executes a series of API requests defined in a JSON file. To speed up the setup, we have added a [fixtures file](fixtures/stripe-fixtures.json) to bootstrap test product and pricing data in your Stripe account. Simply run `stripe fixtures fixtures/stripe-fixtures.json`.
```

> **Important:** Be sure to start the webhook forwarding (see below) so that the products created by the fixtures command above are imported into your database.

Reviewer 2: Shouldn't this come _after_ the webhook section then?

Reviewer: Yes, this should be moved underneath the webhooks setup section

Author: Do you mean this entire section?
